hbase_sql/src/main/java/com/splicemachine/derby/impl/SpliceSpark.java

```diff
@@ -149,6 +149,15 @@ private static SparkSession initializeSparkSession() {
             conf.set("spark.yarn.keytab", HConfiguration.unwrapDelegate().get("hbase.regionserver.keytab.file"));
         }
 
+        // fallback on hbase.master.keytab.file property if we don't yet have a keytab
+        if(conf.get("spark.yarn.principal", "") != ""){
```

Comment:
I don't think it's a problem here, but in general we shouldn't compare strings with the (in)equality operators, use .isEmpty() or .equals() instead.